specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -59,7 +59,7 @@
           "200": {
             "description": "List of `Subscription`s created after the filter.",
             "schema": {
-              "$ref": "#/definitions/Properties"
+              "$ref": "#/definitions/AvailableScopeProperties"
```

Comment:
This was an existing swagger element which was pointing to a non-existent defintion. Fixed it to pass model validation.